samples/gradle.properties

```diff
@@ -24,3 +24,6 @@ avito.logging.verbosity=WARNING
 avito.build=local
 #
 android.useAndroidX=true
+# from common properties
```

Comment:
as I remember we discuss to add `generated when a property generated

Reply:
https://github.com/avito-tech/avito-android/pull/1168#discussion_r689426112